src/smartbanner.js

```diff
@@ -106,11 +107,31 @@ export default class SmartBanner {
   }
 
   get buttonUrl() {
-    if (this.platform === 'android') {
-      return this.options.buttonUrlGoogle;
-    } else if (this.platform === 'ios') {
-      return this.options.buttonUrlApple;
+    let o = this.options;
```

Comment:
Let's not call things `o` or `ä`. We don't want to document code inline so let's keep to fully readable variable names regardless of scope. Could we use `options`?